src/test/java/com/amihaiemil/docker/RtContainersTestCase.java

```diff
@@ -196,4 +196,100 @@ public void ioErrorIfResponseIs500() throws IOException {
             ), URI.create("http://localhost/test")
         ).create("some_image");
     }
+
+    /**
+     * Test for
+     * {@link RtContainers#create(String, String)}: The request URI should be
+     * welformed and include the 'name' query param.
+     * 
```

Comment:
@llorllale could you remove these spaces between the javadoc and the ``@throws``? They look noisy :D

Reply:
@amihaiemil ok